CMake/Assets.cmake

```diff
@@ -142,16 +142,12 @@ set(devilutionx_assets
   ui_art/creditsw.pcx
   ui_art/hf_titlew.pcx
   ui_art/mainmenuw.pcx
-  ui_art/supportw.pcx)
-
-if(VIRTUAL_GAMEPAD)

```

Comment:
`if(NOT USE_SDL1)`?

Reply:
> do you have any hesitations regarding adding the touch assets to SDL1

Yeah, considering that we use SDL1 on some very resource-constrained platforms, let's not add unnecessary stuff to it, even though this is just the MPQ data, dwarfed by `diabdat.mpq` itself.